mixer/adapter/kubernetesenv/config/config.proto

```diff
@@ -67,4 +67,13 @@ message Params {
     //
     // Default: 5 minutes
     google.protobuf.Duration cache_refresh_duration = 2 [(gogoproto.nullable)=false,(gogoproto.stdduration) = true];
+
+    // Namespace for secret created for multi-cluster support.
```

Comment:
Small nit: It would be nice to provide some documentation here (or linked from here) about the format of the secret, and maybe how it is created.